### PR TITLE
serverless plugin to validate graphql schemas using apollo managed federation

### PR DESCRIPTION
**Pull request by Imran99**
## tl;dr
We recently migrated to graphql federation. After implementing this we found that it was possible to cause the graphql gateway to fail at runtime if an implementing service defined an invalid schema. I documented the issue [here](https://github.com/workivate/wellbeing-graphql-api/pull/246). I'm fixing this here.

### Fixing the Apollo Server code
After some digging I found that there is an [open issue for this](https://github.com/apollographql/apollo-server/issues/4499). I didn't have much joy trying to modify the Apollo Server code to do a graceful fallback. I might take another look when I get time.

### Using the existing schema validation provided by Apollo managed federation
If we upload the schema to Apollo managed federation, it will combine it with the schema from all of our implementing services, validate it for us and flag any errors. This way we get build time errors if a schema is broken and it should never make its way to production. Managed federation is free to use so no issues there.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "bugs": {
+    "url": "https://github.com/Imran99/serverless-plugin-apollo-graphql-federation/issues"
+  },
+  "homepage": "https://github.com/Imran99/serverless-plugin-apollo-graphql-federation#readme",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest --config ./jest.config.js"
+  },
+  "dependencies": {
+    "apollo": "^2.30.3"
```
Comment (Perrogramador) [inline package.json]: `lodash.get` and `chalk` are needed

**Review by Perrogramador — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Imran99) [reply package.json]: Good spot thanks!
